README.md

```diff
@@ -228,6 +237,49 @@ PLUGINS_CONFIG = {
 - `token` - (optional) Required for 'Access Token Authorization'.
 - `username` - (optional) Required for 'Secret Server Cloud', 'Password Authorization', 'Domain Authorization'.
 
+### Keeper Secret Manager
+
+The Keeper Secret Manager plugin includes two providers:
+
+- **`Keeper by UID`**
+
+    This provider uses the `UID` to specifiy the secret that is retrieved. The `UID` is displayed in the `Record Info`.
+
+    - Example:
+
+        The url is: _https://keepersecurity.com/vault/#detail/OuiSc5IkgxEBnbnSyVJ5UA_
+
+        In this example the value for `UID` is **OuiSc5IkgxEBnbnSyVJ5UA**.
+
+- **`Keeper by Name/Title`**
+
+    This provider allows to select the secret by name/title.
+
+    - Example:
+
+        The url is _https://keepersecurity.com/vault/#search/XXX-SW-XXX-F0_
+
+        In this example the value for `Title` is **XXX-SW-XXX-F0**.
+
+#### Configuration
+
+```python
+PLUGINS_CONFIG = {
+    "nautobot_secrets_providers": {
+        "keeper": {  # https://github.com/Keeper-Security/secrets-manager/tree/master/sdk/python
+            "token": os.getenv("KEEPER_TOKEN", None),
```

Comment:
Given the listed required/optional keys below, should `type` be added to this example and `token` removed?

Reply:
Type is per secret, Token is more "global"